### PR TITLE
chore: create issue forms and a PR template

### LINKED ISSUE #52: Create issue forms
The main Renovate repository has nice issue forms, so that people who report a bug or request a feature are sure to give all the information we need.

Right now we don't have any issue forms.
Opening a issue to discuss what information we need. 😉

### LINKED ISSUE #53: Create PR template
There is no template given when you open a PR on this repository.
I'd like to have a template. 😄 

This issue is to discuss what kind of information we want to display in the PR template.
Or maybe we just want to copy/paste and adjust the PR template from the main Renovate repository?

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Copy/paste and adjust forms from the main Renovate repository [^main]
- Copy/paste and adjust PR template form the main Renovate repository [^main]
- Configure Prettier to lint/fix `.yml` files
- Block empty issues, all issues need to go through the form interface

## Context:

Closes #52, Closes #53.

Please check if I'm allowed to copy/paste and adjust from the Renovate repository. I don't know if the main Renovate license allows this kind of copy/paste job, even though its within our own organization...

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

[^main]: https://github.com/renovatebot/renovate

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: Note that we will need to do #59 first before we can merge this PR, or the labels won't apply properly. 😄

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]: This description can probably be trimmed down to just `approval bot`?

File: .github/ISSUE_TEMPLATE/config.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/config.yml:1]: This prevents people from opening a empty issues. All issues go through the forms after merging this PR.

File: .github/ISSUE_TEMPLATE/refactor.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/refactor.yml:8]: Is `refactor` a verb? 

I think people use `refactoring` to mean: "the act of cleaning up existing code", so that would be a verb for sure. But I'm not sure if `refactor` on its own is a verb.

I'll let a native English speaker decide. Feel free to make a suggestion to change this. 😉

File: .github/pull_request_template.md
Comment (HonkingGoose) [inline .github/pull_request_template.md:29]: This comment on line 29 "PRs will always be squashed by us when we merge your work" might not be true right now?

Can you check what the current settings are, and maybe change them so that this is accurate? Or else you can suggest a change to the comment. 😉

**Review by viceice — APPROVED** [review]
LGTM

File: .github/pull_request_template.md
Comment (maxbrunet) [reply .github/pull_request_template.md:29]: Yes, squash's best!

File: .github/ISSUE_TEMPLATE/refactor.yml
Comment (maxbrunet) [reply .github/ISSUE_TEMPLATE/refactor.yml:8]: Not a native speaker, but even though it is not in a dictionary, it is a verb for programmers

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (maxbrunet) [reply .github/ISSUE_TEMPLATE/bug_report.yml:9]: As you like, I would suggest to always capitalize it: "Renovate Approve Bot"

https://www.thesaurus.com/e/grammar/when-to-capitalize-words/

**Review by maxbrunet — DISMISSED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (HonkingGoose) [reply .github/ISSUE_TEMPLATE/bug_report.yml:9]:
```suggestion
      description: What version of the Renovate Approve Bot are you using?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:6]:
```suggestion
    id: version-of-renovate-approve-bot
```

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/feature_request.yml:6]:
```suggestion
    id: what-would-you-like-renovate-approve-bot-to-be-able-to-do
```

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]:
```suggestion
      label: What would you like the Renovate Approve Bot to be able to do?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]: @rarkins can you please review this PR, I've copy/pasted and adjusted stuff from the main Renovate repository. Not sure if that's allowed according to the Renovate main repo license. I think you should have final say on whether we merge this or not. 😉

**Review by viceice — APPROVED** [review]
🤷‍♂️

**Pull request merged**
